terraform/modules/pre-auth-lambda/dynamodb.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_dynamodb_table" "dynamodb_table_user" {
+  name         = "user"
```

Comment:
this is quite a vague name, can it be more specific e.g. `analytical-frontend-service-user`? There's a chance we will have other user tables in the future

Reply:
This is done.